### ISSUE TITLE
Design Fuel Instruments

### ISSUE THREAD

**Issue by CaptainBobSim** [CaptainBobSim/The-Cessna-172-Project-V3#146]

**Comment by CaptainBobSim** [comment]
@Kovinator took care of this one!
Might also adjust it for x27 steppers.
Excellent work!

**Comment by Kovinator** [comment]
Yep steppers would be cool! The more options we have as sim builders the better! One thing that’s hard to tell are the left right panels behind the glass or in front of it ? I designed it so the panels go behind the glass. I think it looks good that way but not sure if that’s accurate.

**Comment by CaptainBobSim** [comment]
Hello Kovinator!
Could you send an email to CaptainBobSim@gmail.com with your Fusion 360 username (email) so I can share the team access with you?
